### PR TITLE
Alert utility example

### PR DESCRIPTION
**Pull request by russellnelsonbriggs**
### Description
Provides an example module (alert_util.py) that provides a CLI to summarize Alerts using the SDK

### Issues Resolved
[List any issues this PR will resolve. e.g., Fixes #01]

### Check List
- [NA - example only ] New functionality includes testing.
  - [NA - example only ] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [NA ] New functionality has been documented in the README if applicable.
  - [NA ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [NA ] New endpoints supported are updated in the endpoints-support.md file.
- [ NA] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Comment by russellnelsonbriggs** [comment]
Hi folks, my build failed in Travis with the following that I am not sure what to do about...
Can anyone help me with this?  thank you in advance!

$ pyenv global system 3.6
pyenv: version `3.6' not installed
The command "pyenv global system 3.6" failed and exited with 1 during .

**Comment by russellnelsonbriggs** [comment]
found this.  https://travis-ci.community/t/error-pyenv-version-3-6-not-installed-since-a-week/1400

any chance someone with access to the build env could check into it?  thank you!

**Comment by sijeesh** [comment]
@russellnelsonbriggs, Thank you for your contribution.
Let me try to fix this build issue.

**Comment by sijeesh** [comment]
@russellnelsonbriggs, This issue has been fixed in PR #448 .
Now you can update your branch with the latest changes and push it.

**Comment by russellnelsonbriggs** [comment]
Thank you sir!

File: examples/alert_util.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+        self.client = self.connect(CONFIG)
+
+    def connect(self, config):
+        # Try load config from a file (if there is a config file)
+        config = try_load_from_file(config)
+        return OneViewClient(config)
+
+    def get_alerts(self, _filter='', _view=''):
+        print("\nFilter: {}\n".format(_filter))
+        resp = self.client.alerts.get('/rest/alerts?filter={}'.format(_filter))
```
Comment (sijeesh) [inline examples/alert_util.py:50]: Do we need two get calls in this method? 
Can we use one get_all method with the filter to get all the alerts then print the total count and return the result?

Comment (russellnelsonbriggs) [reply examples/alert_util.py:50]: In this case yes.  The reasoning is:  if you run the get_all() on a large number of alerts (I was testing with 60k), the default page size is only 25!  This causes the get_all to be very, very slow.  So I found that if I give it a count value that is the largest possible (total), it then set the page size to be 1000, which returned my 60k alerts in under a minute.

AND, the only way to get the total number of alerts quickly, without actually trying to fetch them all, was to use the  get().  This way it only gets one page.

Comment (sijeesh) [reply examples/alert_util.py:50]: Ok, got it.

**Review by sijeesh — APPROVED** [review]
LGTM

**Review by AnushaY1916 — APPROVED** [review]

**Pull request merged**